### DISCUSSION TITLE
希望能使输入法状态增加到“中文”、“英文大写”、“英文小写”三种状态

### DISCUSSION THREAD

**Discussion by revinst** [rime/home#1457] [Ideas]
希望能使输入法状态从现在的“中”、“英”两种状态显示，增加到“中文”、“英文大写”、“英文小写”三种状态。可标记为“中”、“A”、“a”等。

**Comment by dufei99** [comment]
那要是要切换到中文，有可能要按2次键，增加了负担，要多个字母大写直接CapsLock就可以吧。觉得不是很有必要

**Comment by dsanke** [comment]
[ImTips](https://github.com/aardio/ImTip) 如果能在rime里实现imtips的功能就好了，输入光标激活激活的时候就显示当前的输入状态，并且可以指示中文/英文/大写锁定以及全角半角

**Reply by dufei99** [reply]
确实，这个看着挺好。
